### DISCUSSION TITLE
Add more builtin operators as resetting keys

### DISCUSSION THREAD

**Discussion by bmrips** [m4xshen/hardtime.nvim#46] [General]
Is there a reason that many builtin operators like `.`, `=`, `~`, `J`, `s`, `<C-a>`, and `g@` are not contained in the list of resetting keys?

**Comment by m4xshen** [comment]
No, I just didn't think of these keys. I'll consider adding them in the future!

**Reply by bmrips** [reply]
Ok! Btw, I meant resetting, not restricting keys.

Wouldn't it be easier to reset timers every time the mode is changed? This would catch any operator that switches to operator-pending, visual, select, or replace mode implicitly. Only operators like `.`, `J`, `<C-a>`, and `~` which stay in normal mode would need to specified explicitly.

**Comment by m4xshen** [comment]
Solved in #51
